helm_deploy/values-preprod.yaml

```diff
@@ -112,3 +112,6 @@ generic-prometheus-alerts:
 
 env_details:
   production_env: false
+
+cas3-report:
+  end-date-override: 3
```

Comment:
I could be wrong on this but if you want to provide config to spring I thought we had to:

1. Define the baseline value in application.yml
2. Define the helm files using the CAPITAL_STYLE e.g. USER-ALLOCATIONS_RULES_LEGACY-ALLOCATOR_ENABLED?

Looking at other lowercased entries in this file they don't appear to be used anywhere in spring as defined?

Reply:
Defining the properties in application.yaml is to set the property to all env as default and you override the value in different env files if you need. In this case we want to test the performance in only pre-prod env and its not applicable to Prod.
The UPPER case property in helm file is to add the property as env variable which you can change without making code change.